src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const rows = table.querySelector('tbody');
+const columnHeaders = table.querySelectorAll('thead th');
+
+columnHeaders.forEach((header, i) => {
```

Comment:
It no critical here (rest of the code here looks good :+1: ), since there aren't a lot of headers, but might be useful in the next tasks (FYI):
https://javascript.info/event-delegation